src/Storage/Device/MinIO.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Utopia\Storage\Device;
+
+use Utopia\Storage\Device\S3;
+
+class MinIO extends S3
+{
+    /**
+     * Regions constants
+     *
+     */
+    const EU_CENTRAL_1='eu-central-1';
+    const US_SOUTHEAST_1='us-southeast-1';
+    const US_EAST_1='us-east-1';
+    const AP_SOUTH_1='ap-south-1';
+
+    /**
+     * Object Storage Constructor
+     *
+     * @param string $root
+     * @param string $accessKey
+     * @param string $secretKey
+     * @param string $protocol
```

Comment:
```suggestion
```

Reply:
Protocol has to be here, because MinIO can be hosted in an internal Network, where you don't have SSL